src/coreclr/jit/emit.h

```diff
@@ -1819,6 +1831,20 @@ class emitter
 
 #endif // TARGET_XARCH
 
+#ifdef TARGET_ARM64
+    struct instrDescLclVarPair : instrDesc // contains 2 gc vars to be tracked
+    {
+        instrDescLclVarPair() = delete;
+
+        emitLclVarAddr iiaLclVar2;
+    };
+
+    struct instrDescLclVarPairCns : instrDescCns // contains 2 gc vars to be tracked, with large cons
+    {
+        emitLclVarAddr iiaLclVar2;
+    };
```

Comment:
What if you instead defined this as:
```suggestion
    struct instrDescLclVarPairCns : instrDescLclVarPair // contains 2 gc vars to be tracked, with large cons
    {
        instrDescLclVarPairCns() = delete;
        
        cnsval_ssize_t idcCnsVal;
    };
```

That would simplify all the code like:
```
        if (id->idSmallCns())
        {
            ((instrDescLclVarPair*)id)->iiaLclVar2.initLclVarAddr(varx2, offs);
        }
        else
        {
            ((instrDescLclVarPairCns*)id)->iiaLclVar2.initLclVarAddr(varx2, offs);
        }
```
=>
```
        instrDescLclVarPair* id = emitNewInstrLclVarPair(attr, imm);
        id->iiaLclVar2.initLclVarAddr(varx2, offs);
```

Reply:
sure, that would be simpler.